docs/release_notes.rst

```diff
@@ -46,6 +46,53 @@ New Platforms
 
 RTI has validated that *Connector* can be used on macOS 11 (Big Sur) systems.
 
+
+New API makes it easier to query what version of Connector is being used
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. CON-92 
+
+A new API, :meth:`rticonnextdds_connector.Connector.get_version`, has been added that provides the caller
+with the version of *Connector* and the version of the native libraries being used.
+
+
+What's Fixed in 1.2.0
+---------------------
+
+Error logged when accessing string longer than 128 bytes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Previously, on an input, when accessing a string longer than 128 bytes, the
+following error was printed:
+
+.. code-block::
+
+    Output buffer too small for member (name = "frame", id = 1). Provided size (128), requires size (x).
+
+This error message was innocuous; there was actually no issue with retrieving
+the string. The message is no longer printed.
+
+[RTI Issue ID CON-157]
+
+
+Deleting same Connector object twice may have resulted in segmentation fault
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+A segmentation fault may have occurred when the same *Connector* object was
+deleted twice. This issue has been resolved.
+
+[RTI Issue ID CON-209]
+
+
+Support added for handling large 64-bit integers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Support has been improved for both getting and setting large (greater than 2^53)
+64-bit values. See :ref:`section-access-64-bit-integers` for more information.
+
+Note that on Windows systems, the string representations of Not a Number and infinity
+(e.g., ``'NaN'``, ``'Infinity'``) are not valid values for a Number. They are valid
+inputs on other systems.
+
+[RTI Issue ID CON-190]
+
+
 Previous Releases
 ~~~~~~~~~~~~~~~~~
 
```

Comment:
@samuelraeburn, what happened to our Version 1.1.1 section in Previous Releases?? It just disappeared from the release branch. Ugh!

8.4.1. Version 1.1.1
Connector 1.1.1 is built on RTI Connext DDS 6.1.0.3, which fixes several bugs in the Core Libraries. If you want more details on the bugs fixed in 6.1.0.3, contact support@rti.com. These bugs are also fixed in RTI Connext DDS 6.1.1, upon which RTI Connector 1.2.0 is built.

Reply:
Oh, I forgot to merge the original release notes PRs to the release branch before creating this one. I don't know how this merge will go, then.

Reply:
I merged the release branch back into this one, and I think that made things right.